models/prices/prices_tokens.sql

```diff
@@ -232,6 +232,7 @@ VALUES
     ("chsb-swissborg", "ethereum", "CHSB", "0xba9d4199fab4f26efe3551d490e3821486f135ba", 8),
     ("chz-chiliz", "ethereum", "CHZ", "0x3506424f91fd33084466f402d5d97f05f8e3b4af", 18),
     ("cnd-cindicator", "ethereum", "CND", "0xd4c435f5b09f855c3317c8524cb1f586e42795fa", 18),
+    ("cndl-candle", "ethereum", "CNDL", "0x3D97EdB1c1D87C0cBf098a0D2230d7380d4b1432", 18),
```

Comment:
@andreistefanwork can you please make the contract address value lowercase?

Reply:
also, looks like this token is inactive (note the is_active key): https://api.coinpaprika.com/v1/coins/cndl-candle -- you can look to activate through coinpaprika directly: https://coinpaprika.com/add/